tests/sentry/tasks/test_delete_seer_grouping_records.py

```diff
@@ -108,23 +79,55 @@ def test_call_delete_seer_grouping_records_by_hash_chunked(self) -> None:
     @patch(
         "sentry.tasks.delete_seer_grouping_records.delete_seer_grouping_records_by_hash.apply_async"
     )
-    def test_call_delete_seer_grouping_records_by_hash_no_hashes(
-        self, mock_apply_async: MagicMock
-    ) -> None:
-        self.project.update_option("sentry:similarity_backfill_completed", int(time()))
+    def test_called_task_with_too_many_hashes(self, mock_apply_async: MagicMock) -> None:
+        """This tests the built-in logic of spreading hashes across multiple tasks."""
+        batch_size = 10
+        with self.options({"embeddings-grouping.seer.delete-record-batch-size": batch_size}):
+            # Create 15 group hashes to test chunking (10 + 5 with batch size of 10)
+            _, expected_hashes = self._setup_groups_and_hashes(batch_size + 5)
+            # Call function directly rather than scheduling a task
+            delete_seer_grouping_records_by_hash(self.project.id, expected_hashes, 0)
```

Comment:
@markstory @JoshFerge This shows the fix for #95156 more clearly.
We now test that the task with too many hashes gets spread out.